frontend/src/components/views/MapView.jsx

```diff
@@ -2,7 +2,7 @@ import { convertToRenderCoordinates, tileConfig } from "../../utils/render_utils
 
 export default function MapView({assets, gameMap, buildTower, newTower})
 {
-    function renderTile(tile) {
+    function renderTile(tile, _coordinates) {
```

Comment:
What is `_coordinates` for? I don't see it being used.

Reply:
The underscore is typically a way to indicate the variable isn't being used.

This is a common situation when you have an interface that pass multiple variables but you don't necessarily need all of them. A good example is [Map.forEach()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/forEach), which passes three variables to the callback  provided by the user but you often don't need all of them.

As to why I implemented `GameMap` that way, I think it's possible we'll one day have a client of the class that uses it. However, it is a bit redundant given how we currently have tile defined but that isn't the fault of GameMap class itself.